test/edu-benefits/5490/config/benefitHistory.unit.spec.jsx

```diff
@@ -57,7 +57,7 @@ describe('Edu 5490 benefitHistory', () => {
 
     // Should expand to 16
     expect(Array.from(formDOM.querySelectorAll('input,select')).length)
-      .to.equal(16);
+      .to.equal(17);
```

Comment:
There might be a similar test somewhere else, but lines 330 and 337 in the config above are new and wouldn't be covered by a test yet.